src/highlighted-card.tsx

```diff
@@ -76,6 +76,7 @@ const Content = React.forwardRef<HTMLDivElement, Props>((props, ref) => {
             className={styles.container}
             dataAttributes={{'component-name': 'HighlightedCard', ...props.dataAttributes}}
             width={props.width ? `${props.width}px` : '100%'}
+            minHeight="100%"
```

Comment:
Highlighted card was not adapting it's height to the container (like all other cards do).